pkg/kconfig/kconfig.go

```diff
@@ -337,6 +337,10 @@ func (kp *kconfigParser) parseProperty(prop string) {
 		kp.ConsumeLine()
 	case "modules":
 	case "optional":
+	// transitional is used for the configs backward compatibility.
```

Comment:
".. for config backwards compatibility."

Reply:
https://github.com/google/syzkaller/pull/6394 to address it.